DQM/L1TMonitor/src/L1TStage2uGT.cc

```diff
@@ -147,6 +168,73 @@ void L1TStage2uGT::bookHistograms(DQMStore::IBooker &ibooker, edm::Run const&, e
    prescaleFactorSet_ = ibooker.book2D("prescaleFactorSet", "uGT: Index of Prescale Factor Set vs. LS", numLS, 0., numLS_d, 25, 0., 25.);
    prescaleFactorSet_->setAxisTitle("Luminosity Segment", 1);
    prescaleFactorSet_->setAxisTitle("Prescale Factor Set Index", 2);
+
+  // Prescaled and Unprescaled Algo Trigger Bits
+  prescaled_first_collision_run_ = ibooker.book2D("prescaled_first_collision_run_", "uGT: Prescaled Algorithm Trigger Bits (First bunch) vs. BX Number In Event", 5, -2.5, 2.5, numAlgs, -0.5, numAlgs_d-0.5);
+  prescaled_first_collision_run_->setAxisTitle("Bunch Crossing Number In Event", 1);
+  prescaled_first_collision_run_->setAxisTitle("Algorithm Trigger Bits+Names (First bunch in train)", 2);
+  for(unsigned int algo=0; algo<prescaledAlgoBit_.size(); algo++) {
+    prescaled_first_collision_run_->setBinLabel(prescaledAlgoBit_.at(algo),prescaledAlgoShortList_.at(algo)+"("+std::to_string(prescaledAlgoBit_.at(algo))+")",2);
```

Comment:
The bin index and the algo bit are not the same thing. The first argument of setBinLabel should be the bin index and not the algo bit.